promise-it-wont-hurt-solutions-master/src/08-values-and-promises.js

```diff
@@ -0,0 +1,20 @@
+
+// var promise = new Promise(function () {
+//     attachFile( () => {
+//       let attachFile = 'DR. ';
+//       fulfill('MANHATTAN');
+//     });
+//     });
+//     promise.then(console.log(attachFile + 'MANHATTAN'));
+'use strict';
+const attachFile = 'DR. ';
+var promise = new Promise(function (fulfill, reject) {
+
+
+    return attachFile('DR. ') || fulfill('MANHATTAN');
```

Comment:
attachFile() is not defined as a function, is what I meant to write.

This code works:
function attachTitle(string) {
  return 'DR. ' + string;
}

const promise = Promise.resolve('MANHATTAN');

promise.then(attachTitle).then(console.log);